DeviceBridgeE2E/data/deviceTemplates/basic.json

```diff
@@ -0,0 +1,114 @@
+{
+    "capabilityModel": {
+        "@context": [
+            "http://azureiot.com/v1/contexts/IoTModel.json",
+            "http://azureiot.com/v1/contexts/solutionmodel.json",
+            "http://azureiot.com/v1/contexts/modeldefinition.json"
+        ],
+        "@id": "dtmi:jeff:Basic9;1",
+        "@type": "CapabilityModel",
+        "contents": [
+            {
+                "@id": "dtmi:jeff:Basic8:temperature;1",
```

Comment:
Can we replace the references to `jeff` by something else in this file? :D
(just replace the occurences by something like `test`)